packages/ProgressAccordion/src/ProgressAccordion.js

```diff
@@ -63,7 +63,7 @@ const ProgressAccordion = props => {
   };
 
   return (
-    <div css={accordionStyles} {...moreProps} role="list" aria-label={a11yText}>
+    <div css={accordionStyles} {...moreProps} role="list" aria-label={a11yText} data-pka-anchor="progress-accordion">
```

Comment:
Btw... is this not an `<ul>` for global styles reasons?

Reply:
I would think that would be the reason.

Reply:
Right.  The `listitem` is also a `<div>` instead of an `<li>` too.  It's more _resilient_ this way.